op-e2e/faultproofs/output_alphabet_test.go

```diff
@@ -104,11 +104,11 @@ func TestOutputAlphabetGame_ValidOutputRoot(t *testing.T) {
 func TestChallengerCompleteExhaustiveDisputeGame(t *testing.T) {
 	op_e2e.InitParallel(t, op_e2e.UseExecutor(1))
 
-	testCase := func(t *testing.T, isRootCorrect bool) {
-		ctx := context.Background()
-		sys, l1Client := startFaultDisputeSystem(t)
-		t.Cleanup(sys.Close)
+	ctx := context.Background()
+	sys, l1Client := startFaultDisputeSystem(t)
+	t.Cleanup(sys.Close)
```

Comment:
I'm not sure of the exact go test life cycle but I worry a little bit that we may wind up spinning up this system, then the test pauses when we call `t.Run` and the system is left running while we're queued waiting on other tests.

Reply:
Cleanup operates on the test or subtest level: https://pkg.go.dev/testing#T.Cleanup

Reply:
I'm more worried about the timing.  ie it runs this and starts the system, then the `t.Run` calls queue up the subtests, but potentially those subtests don't run immediately and are queued. Meanwhile the system is still running in the background waiting for the subtests to complete.